examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<ul><li><code>first_derivative(pn, dim=y)</code>  are these debug leftovers?</li><li><code>u.data[:] = 0.</code>  and such are not necessary, initialized to zero by default</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
>     * `first_derivative(pn, dim=y)`  are these debug leftovers?
Yes. I didn't know you could do p.dx or p.dxc when I started. I will remove these.
>     * `u.data[:] = 0.`  and such are not necessary, initialized to zero by default
I didn't know this happened